source/includes/steps-install-bi-connector.yaml

```diff
@@ -83,7 +83,7 @@ action:
          .. cssclass:: copyable-code
          .. code-block:: powershell
 
-            move <donwloadDirectory>/bin/* "C:\Program Files\MongoDB BI Connector"
+            move {downloadDirectory}\* "C:\Program Files\MongoDB BI Connector"
```

Comment:
Is there a reason you used curly braces here? I thought our house style was angle brackets